assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -908,6 +938,29 @@ describe("DiversionPage", () => {
     ).toBeVisible()
   })
 
+  test("When 'Review Detour' button is clicked, shows 'Share Detour Details' screen but there was an error", async () => {
```

Comment:
I think we should make the test name a bit clearer.
```suggestion
  test("When API call errors and 'Review Detour' button is clicked, shows 'Share Detour Details' screen", async () => {
```